internal/format/asciidoc_table_test.go

```diff
@@ -136,6 +136,7 @@ func TestAsciidocTableNoHeader(t *testing.T) {
 		ShowOutputs:      true,
```

Comment:
`TestAsciidocTableNoResources` and `TestAsciidocTableOnlyResources` functions are missing.